lib/importer.py

```diff
@@ -852,6 +889,33 @@ def execImport(handle: int, options: dict):
         log(f"cannot retrieve {mediaTypes} items without any library section", xbmc.LOGERROR)
         return
 
+    # Decide if doing fast sync or not, if so set filter string to include updatedAt
+    fastSync = True
+    lastSync = mediaImport.getLastSynced()
+
+    # Check if import settings have changed, or if this is the first time we are importing this library type
+    if not lastSync:
+        fastSync = False
+        SynchronizationSettings.CalculateHash(
+            importSettings=importSettings,
+            providerSettings=providerSettings,
+            save=True
+        )
+        log("first time syncronizing library, forcing a full syncronization", xbmc.LOGINFO)
+    elif SynchronizationSettings.HaveChanged(
+            importSettings=importSettings,
+            providerSettings=providerSettings,
+            save=True
+    ):
+        fastSync = False
+        log("library import settings have changed, forcing a full syncronization", xbmc.LOGINFO)
+
+    if fastSync:
+        log(f"performing fast syncronization of items viewed or updated since {str(lastSync)}")
+        lastSyncEpoch = parser.parse(lastSync).strftime('%s')
+        updatedFilter = {'updatedAt>': lastSyncEpoch}
+        watchedFilter = {'lastViewedAt>': lastSyncEpoch}
```

Comment:
```suggestion
        updatedFilter = {"updatedAt>": lastSyncEpoch}
        watchedFilter = {"lastViewedAt>": lastSyncEpoch}
```

Reply:
What is your thought for this? Unofficial convention is single quotes for dictionary keys and all other dicts I found in the code (self._state in player.py for example) use single quotes for their keys.

Reply:
I'm not a proficient python developer to have an actual opinion on this ;-)

Reply:
It's pretty all over the place with python and varies by developer and use of the two isn't defined in PEP8 to clear it up.

The main convention I have seen is single-quote's for object-y references (dict keys mostly), singles when passing a short string argument in, and double for pretty much everything else so that single's can be used inside them. With triple-doubles for block comments and docstrings, never triple-singles.

I think all the dicts that I found were commits by enen. Throughout the file there is a mix of single and double for the long strings (probly a mix of you and enen :P)